generate-genesis.js

```diff
@@ -7,6 +7,22 @@ const web3 = require("web3")
 const validators = require("./validators")
 const init_holders = require("./init_holders")
 
+
+program.option("--bscChainId <bscChainId>",
+    "bscChainId",
+    "0060");
+program.option("-c, --chainid <chainid>", "chain id", "714")
```

Comment:
Do we need to specify this parameter in generating command?

Reply:
No need

Reply:
This option maybe useless, let me double check

Reply:
ok, this affect:
![image](https://user-images.githubusercontent.com/7310198/106848227-83d42c00-66eb-11eb-9639-319f173d4ed6.png)
do not affect the bytes code. I will fix this as well